Vladyslav-Hudz/README.md

```diff
@@ -12,33 +14,61 @@ Before getting started, you will need:
 
 ## Step 1: Installing Arduino IDE
 
-1. Visit the official Arduino website at [arduino.cc](https://www.arduino.cc/).
+1. Visit the official Arduino website at [arduino.cc/software](https://www.arduino.cc/en/software).
 2. Download the Arduino IDE from the "Downloads" page.
-3. Install the Arduino IDE on your computer by following the instructions for your operating system.
+3. Install the Arduino IDE on your computer by following the [instructions](https://docs.arduino.cc/software/ide-v2/tutorials/getting-started/ide-v2-downloading-and-installing) for your operating system.
 
 ## Step 2: Connecting Arduino
 
-1. Connect Arduino to your computer using the USB cable.
-2. Launch the Arduino IDE.
+1. Launch the Arduino IDE.
+2. Connect Arduino to your computer using the USB cable.
 
 ## Step 3: Selecting the Board and Port
 
-1. In the `Menu Bar` in the `Tools` menu, select the appropriate Arduino board under `Board`.
-2. Choose the correct `Port` for your Arduino board in the `Tools` menu. If you're unsure which port to select, refer to the documentation specific to your board.
+1. In the `Menu Bar` select `Tools` > `Board` > `Arduino AVR Boards` select the appropriate board.
+
+<center>
+<img src="picture/menu bar.png" title="Upload button" width="600">
+</center>
+
+2. In the `Menu Bar` select `Tools` > `Port`  choose the correct port for your Arduino board. If you're unsure which port to select, refer to the documentation specific to your board.
 
-![The view of the Arduino IDE](https://images.theengineeringprojects.com/image/main/2018/10/Introduction-to-Arduino-IDE-9.jpg)
+<center>
+<img src="picture/port.png" title="Upload button" width="600">
+</center>
+
+>You can find more information about board on [arduino/boards](https://www.arduino.cc/en/hardware#boards)
+>
+>If your board does not appear on a port in Arduino IDE: [arduino/support](https://support.arduino.cc/hc/en-us/articles/4412955149586-If-your-board-does-not-appear-on-a-port-in-Arduino-IDE)
 
 ## Step 4: First Project
 
-Now you're ready to write your first code for Arduino.
+### You can use already existing code examples
+
+1. To view available code examples, select `File` > `Examples` to browse through the available code examples.
+
+<center>
+<img src="picture/example.png" title="Upload button" width="500">
+</center>
 
-1. Select `File` > `Examples` to browse through the available code examples.
-2. Choose one of the examples to open it in the Arduino IDE.
+2. Choose one of the examples to open it in the new Arduino IDE window.
 3. Familiarize yourself with the code and the accompanying comments that explain different parts of the program.
-4. Click the `Upload` button to upload the code to the Arduino board.
+
+### Or write your own code
```

Comment:
Headings should be consistent across the document. This heading uses sentence-style capitalization (which is the recommended best practice) while most of the other headings use title-style capitalization.
For more information on headings, see [Capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization#sentence-style-capitalization-in-titles-and-headings) and [Headings](https://learn.microsoft.com/en-us/style-guide/scannable-content/headings).